plugin/calendar.py

```diff
@@ -0,0 +1,18 @@
+from discord.ext.commands import Bot
+from discord.ext.commands import Cog
+from discord.ext.commands import Context
+from discord.ext.commands import command
+from logic.calendar import CalendarImpl
+
+class Calendar(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+
+    @command()
+    async def add_event(self, ctx: Context, arg: str) -> None:
+        result = CalendarImpl.add_event(arg)
+        await ctx.send(result)
```

Comment:
```suggestion
    async def add_event(self, ctx: Context, title: str, date: str, location: str) -> None:
        CalendarImpl.add_event(title, date, location)
        await ctx.send("Success!")
```
- 後で空白で split するなら、ここに処理を持たせるべき
- 登録出来なかったら add_event の中でエラーが発生するので、こちらでは常に成功のメッセージを返すようにしたら良い